index.html

```diff
@@ -2986,13 +2983,17 @@ <h3>application/ld+json</h3>
           and without knowledge of a profiled resource can safely use the same
           representation.</p>
         <dl>
-          <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
-          <dd>To specify a JSON-LD <a>Frame</a>.</dd>
+          <dt><code>http://www.w3.org/ns/json-ld#frame</code></dt>
+          <dd>To request or specify a <a data-lt="frame">JSON-LD Frame document</a>.</dd>
         </dl>
-        <p>The <code>http://www.w3.org/ns/json-ld#framed</code> SHOULD
-          be used when serving and requesting a JSON-LD frame document.</p>
+        <p>The <code>http://www.w3.org/ns/json-ld#frame</code> `profile` parameter SHOULD
```

Comment:
as the parameter is now marked as 'required', shouldn't SHOULD be changed to MUST?

Reply:
Except, in practice, no one does this :). It was described before as a required parameter, and if it was optional, there would be no need for an IANA section at all. I think SHOULD better reflects the reality of the situation, even if it is somewhat at conflict with “required”.

Reply:
Ok, I can live with that :)